packages/libs/src/sdk/api/challenges/types.ts

```diff
@@ -0,0 +1,94 @@
+import { z } from 'zod'
+import { HashId } from '../../types/HashId'
+
+// @ts-ignore:next-line ignore the unused import, used in jsdoc
+import type { ChallengesApi } from './ChallengesApi'
+
+export enum ChallengeId {
```

Comment:
Decided on enum here to mask the implementation names being short and undecipherable. Check that the names make sense for third parties?

Reply:
Agreed, the short names are very difficult to work with.